src/client/components/card/SeasonPodium.jsx

```diff
@@ -1,22 +1,86 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
 import { PodiumParticipant } from './PodiumParticipant';
 
-export const SeasonPodium = () => {
+import '../styling/competition.css';
 
-    return (
-        <div className="podium season-podium">
+export const SeasonPodium = (props) => {
+    const { season } = props;
+
+    const { participants, positionMappings, rounds } = season;
+
+    const [seasonStandings, setSeasonStandings] = useState([]);
+
+    //     for each participant,
+    //     find team,
+    //     find position per round,
+    //     build array,
+    //     sort by points
+    //     map array
+
+    useEffect(() => {
+        let standings = participants.map((participant) => {
+            let participantPositions = [];
+
+            participant.placements.map((placement) => participantPositions.push(placement.position));
+
+            let pointsTotal = 0;
+
+            participantPositions.map((position) => {
+                let pointsForRound = positionMappings[position - 1].mapping;
 
-        <PodiumParticipant />
+                if (isNaN(pointsForRound)) {
+                    pointsForRound = 0;
+                }
 
-        <div className="round-breakdown">
-            {/* This gets mapped; participant's placement for 4 most recent rounds */}
-            <h4>1</h4>
-        </div>
+                pointsTotal = pointsTotal + pointsForRound;
+            });
 
-        <div className="participant-score">
-            <h3>127</h3>
-        </div>
-    </div>
-    )
-}
\ No newline at end of file
+            const pointsTotalforParticipant = {
+                participant: participant,
+                points: pointsTotal,
+            };
+
+            return pointsTotalforParticipant;
+        });
+
+        standings.sort((a, b) => b.points - a.points);
+
+        setSeasonStandings(standings);
+    }, [season]);
+
+    //     for each participant, get position per round
+
+    const generateDataForRounds = (standing) =>
+        rounds.map((round) => round.placements.filter((placement) => placement.participantId === standing.participant.id));
+
+    return (
+        <>
+            {seasonStandings.length > 0 && (
```

Comment:
You're safe to treat length like a bool here, `seasonStandings.length && (` would work fine